LATEST

```diff
@@ -1 +1 @@
-6f8c3ad832ee0fa0c234c30d7312f73e703760f4 0.13.55
+6ea118d6142d2a937286b0a7bf9846dbcdb1751b 0.13.55-snapshot.20200318.3529.0.6ea118d6
```

Comment:
We need to bump the version number as well:
```suggestion
6ea118d6142d2a937286b0a7bf9846dbcdb1751b 0.13.56-snapshot.20200318.3529.0.6ea118d6
```

Reply:
Ok, I've been advised that we edit this manually so as soon as we've decided on which of 0.13.56, 0.14.0 or 1.0.0 I'll make the edit.